aws-codeguruprofiler-profilinggroup/src/test/java/software/amazon/codeguruprofiler/profilinggroup/CreateHandlerTest.java

```diff
@@ -75,24 +82,84 @@
             .actionGroup(AGENT_PERMISSIONS)
             .principals(principals)
             .build();
+    private final AddNotificationChannelsRequest addNotificationChannelsRequest = AddNotificationChannelsRequest.builder()
+            .profilingGroupName(profilingGroupName)
+            .channels(Channel.builder()
+                    .uri("channelUri")
+                    .eventPublishers(ImmutableSet.of(EventPublisher.ANOMALY_DETECTION))
+                    .build())
+            .build();
```

Comment:
Since this is only used within `@Nested class WhenAnomalyDetectionNotificationConfigurationIsSet`, I suggest moving this inside that class, so that it's clear that it's only used in that scope.